### DISCUSSION TITLE
Scrollbar Styling Feature

### DISCUSSION THREAD

**Discussion by yasindunethmina** [tailwindlabs/tailwindcss#10681] [Ideas]
I think tailwind should have a way to style scrollbar, Specially scrollbar color, width & radius at least. Please let me know if this is interesting, I'd like to work on this feature!

Example  CSS code:
*scroll-bg-red-500
*scroll-rounded-sm
*scroll-w-2
*scroll-thumb-red-500
*hover:scroll-red-500

**Comment by lukewarlow** [comment]
In case anyone comes across this discussion looking for a solution I've made an npm package with utility plugins for scrollbar-color, scrollbar-width and scrollbar-gutter. See https://github.com/lukewarlow/tailwind-scrollbar-utilities

This doesn't allow for styling such as rounded as that is only (if at all) possible using the deprecated ::-webkit-scrollbar CSS. Which isn't supported in Firefox. I do however use this legacy CSS for scrollbar-none to improve browser support.
